dracut.conf.d/50-generic.conf

```diff
@@ -0,0 +1,3 @@
+# generic initrd works on other platforms than the one dracut was called
+
```

Comment:
I would remove the empty line between the comment and the config value.

Reply:
ok, thanks.